ocaml/idl/datamodel_vm.ml

```diff
@@ -1514,6 +1514,19 @@ let set_appliance =
       ]
     ~allowed_roles:_R_POOL_OP ()
 
+let set_groups =
+  call ~name:"set_groups" ~in_product_since:rel_boston
+    ~doc:"Associate this VM with a VM group."
+    ~params:
+      [
+        (Ref _vm, "self", "The VM")
+      ; ( Set (Ref _vm_group)
+        , "value"
+        , "The VM groups to set (Only one anti-affinity group is supported now)"
```

Comment:
I think the "(Only one anti-affinity group is supported now)" is too specific to be placed in this description.